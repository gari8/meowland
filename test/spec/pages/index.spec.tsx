import Top from "../../../src/pages/index";
import { render } from "@testing-library/react";

describe("index", (): void => {
  test("Top", () => {
    const { asFragment } = render(<Top />);
    expect(asFragment()).toMatchSnapshot();
  });
});
