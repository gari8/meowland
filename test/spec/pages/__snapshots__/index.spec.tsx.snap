// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index Top 1`] = `
<DocumentFragment>
  <div>
    <p>
      aaa
    </p>
  </div>
</DocumentFragment>
`;
